### ISSUE TITLE
mmctl system version has unspecified and inconvenient format (and mattermost version will be deprecated)

### ISSUE THREAD

**Issue by gavinbeatty** [mattermost/mmctl#414]
Given that [mattermost version will be deprecated](https://docs.mattermost.com/manage/command-line-tools.html?highlight=version#mattermost-version) in favor of [mmctl system version](https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-system-version), the output format of the latter should be intelligible.

However, it is currently just "Server version " followed by the contents of the "X-Version-ID" header, which is filled in [mattermost-server/web/handlers.go](https://github.com/mattermost/mattermost-server/blob/27559c1c7b395dee20c43c310225b24694788d2f/web/handlers.go#L211), i.e.,

````go
fmt.Sprintf("%v.%v.%v.%v", model.CurrentVersion, model.BuildNumber, c.App.ClientConfigHash(), c.App.Srv().License() != nil)
````

Meaning the full output is something like `Server version 5.39.2.5.39.2.b8d48854a28e225558740e00d1dfb8e2.true`. This isn't easy to read, or particularly easy to machine parse _if_ we ever expect version numbers to have more or less than 3 parts.

In particular I would like to extract `5.39.2` and `true/false` (for whether it is the enterprise edition) in a script, as part of an internal maintenance process.

PS: Off-topic, but I'd also like to note that it is quite inconvenient to have to use mmctl to get the binary version information, which should be available without having a running instance.

**Comment by isacikgoz** [comment]
Hey @gavinbeatty, there was a miscommunication about `mattermost version` being deprecated. We did not deprecate the command, in fact we removed the database initialization part from it to just serve getting version purpose. `mmctl system version` is still be improved anyway. But it's another issue I guess?

But the `mattermost version` will return the values in the form of:
```
Version: <semantic_version>
Build Number: <build_number>
Build Date: <build_date>
Build Hash: <build_hash>
Build Enterprise Ready: <true|false>
```
